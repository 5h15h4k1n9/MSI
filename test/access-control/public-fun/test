setValue(123)
value
exit
