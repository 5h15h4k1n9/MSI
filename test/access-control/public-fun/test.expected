(void) void
(int) 123
Bye-bye!
